update_fork.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+git fetch upstream
+git checkout master
+git push origin master
```

Comment:
I believe this file is not necessary. You can easily use alias to achieve the same goal.

Reply:
True, removing.